### ISSUE TITLE
does apisix-dashboard  support deploy multiple node as loadbalance?

### ISSUE THREAD

**Issue by mygrsun** [apache/apisix-dashboard#2391]
### Issue description

I deploy two nodes.Every node has apisix-dashboard and  apisix and I use a host point to the two nodes.But the porblem is I can't login the dashboard.How can I realize dashboard loadbalance?

### Expected behavior

login success

### How to Reproduce

deploy two nodes.Every node has apisix-dashboard and  apisix
 use a host point to the two nodes

### Environment

- apisix version (cmd: `apisix version`):
- OS (cmd: `uname -a`):
- OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
- etcd version, if have (cmd: run `etcd --version`):
- apisix-dashboard version, if have:
- Browser version, if have:

**Comment by guoqqqi** [comment]
> But the porblem is I can't login the dashboard

Can you provide some more detailed steps? What version of the console do you have and which way it is running

**Comment by zaunist** [comment]
@mygrsun. It is not support to manage multiple apisix node for now. So if we have an apisix that needs a dashboard to manage.

**Comment by Baoyuantop** [comment]
Hi @mygrsun, I'm wondering if the two nodes are just load balancing? If so they are configured from the same etcd, then we can use one Dashboard to manage?

**Comment by mygrsun** [comment]
> 

yes ,the two dashboard use zhe same etcd。In fact，I use k8s deploy the node。Every nodes has dashboard-apisix and apisix。

**Comment by mygrsun** [comment]
> @mygrsun. It is not support to manage multiple apisix node for now. So if we have an apisix that needs a dashboard to manage.

I use k8s deply 。So can't I deploy two pods？

**Comment by mygrsun** [comment]
> > But the porblem is I can't login the dashboard
> 
> Can you provide some more detailed steps? What version of the dashboard do you have and which way it is running

I use k8s deploy,dashboard version is 2.10.1

**Comment by Baoyuantop** [comment]
> > 
> 
> yes ,the two dashboard use zhe same etcd。In fact，I use k8s deploy the node。Every nodes has dashboard-apisix and apisix。

If it's the same etcd, why not use one Dashboard?

**Comment by mygrsun** [comment]
> 



> > > 
> > 
> > 
> > yes ,the two dashboard use zhe same etcd。In fact，I use k8s deploy the node。Every nodes has dashboard-apisix and apisix。
> 
> If it's the same etcd, why not use one Dashboard?

if i use only one dashboard ，how to avoid it crash。I want use two node as high reliability

**Comment by guoqqqi** [comment]
cc @nic-chen PTAL ~

**Comment by nic-chen** [comment]
@mygrsun 
Actually, I don't understand what your problem is. 
Are your two dashboard configurations the same? If it is exactly the same, there should be no login problems.

**Comment by juzhiyuan** [comment]
On my side, @mygrsun wants to know if there has a `High availability solution`.

**Comment by bzp2010** [comment]
The UI will again be part of APISIX, so if APISIX is highly available, it will be highly available. Separate deployment is no longer within the scope of this project. ref #2981 

If you have any questions, feel free to start a new issue!
